source/vulkan/runtime_vk.cpp

```diff
@@ -160,7 +179,7 @@ VkImageView reshade::vulkan::runtime_vk::create_image_view(VkImage image, VkForm
 {
 	VkImageAspectFlags aspect = VK_IMAGE_ASPECT_COLOR_BIT;
 	if (format >= VK_FORMAT_D16_UNORM_S8_UINT && format <= VK_FORMAT_D32_SFLOAT_S8_UINT)
-		aspect = VK_IMAGE_ASPECT_STENCIL_BIT;
+		aspect = VK_IMAGE_ASPECT_STENCIL_BIT | VK_IMAGE_ASPECT_DEPTH_BIT;
```

Comment:
This is missing some formats, because the `format >= VK_FORMAT_D16_UNORM_S8_UINT && format <= VK_FORMAT_D32_SFLOAT_S8_UINT` range only covers formats with stencil, but there are valid depth formats without stencil too. So to fix there should be a separate if statement that checks for depth-only formats:
```
if (format >= VK_FORMAT_D16_UNORM_S8_UINT && format <= VK_FORMAT_D32_SFLOAT_S8_UINT)
	aspect = VK_IMAGE_ASPECT_DEPTH_BIT | VK_IMAGE_ASPECT_STENCIL_BIT;
else if (format == VK_FORMAT_D16_UNORM || format == VK_FORMAT_D32_SFLOAT)
	aspect = VK_IMAGE_ASPECT_DEPTH_BIT;
```

Reply:
Ok, will check this ;-)